### ISSUE TITLE
Need a space for an error message

### ISSUE THREAD

**Issue by seokho-son** [cloud-barista/cb-tumblebug#766]
**What happened**
: 

Need a space for the error message in https://github.com/cloud-barista/cb-tumblebug/blob/fc375a6623fb07c0512fb1eb6bae61ef3fbc2ebf/src/core/mcis/control.go#L483

**Proposed solution**
: 

```diff
		result, err := RunSSH(vmIp, sshPort, verifiedUserName, privateKey, cmd)
		if err != nil {
			fmt.Println("[ERR: result] " + "[ERR: err] " + err.Error())
-			return "", "", fmt.Errorf("Cannot do ssh, with" + verifiedUserName + ", " + err.Error())
+			return "", "", fmt.Errorf("Cannot do ssh, with " + verifiedUserName + ", " + err.Error())
		}
```

with" --> with "

**Comment by betelgeuse-7** [comment]
`return "", "", fmt.Errorf("cannot do ssh with %s, %s", verifiedUserName, err.Error())`

**Comment by seokho-son** [comment]
Hello @betelgeuse-7 

Are you interested on this issue ? :) 
If so, I can assign this issue to you.

This issue is easy to handle and we are happy to assist you !

**Comment by betelgeuse-7** [comment]
@seokho-son yes, I can take this issue. This is easy to solve. But I cannot make a pr today, but tomorrow. If that's okay to you, then assign me. :)

**Comment by seokho-son** [comment]
@betelgeuse-7 You don't have to rush at all :)
I am happy to assign new contributor !

**Comment by betelgeuse-7** [comment]
ok thanks @seokho-son

**Comment by seokho-son** [comment]
Fixed from the PR #780
